scripts/gen_launch.py

```diff
@@ -3,103 +3,109 @@
 # USAGE: after changing tests, generate a launch.json configuration with this script
 # ./scripts/gen_launch.py > .vscode/launch.json
 
-import json;
-import copy;
-import subprocess;
+import json
+import copy
+import subprocess
 from typing import Final
 
+
 def get_launch_json_examples(input: str, crate: str) -> list[dict]:
```

Comment:
What are your thoughts on updating the type information for the containers as well? in this case, the type `list[dict]` would use the more conformant `List[Dict[A, B]]` where `A` and `B` are placeholders for what the real types are (looks like `str, str` would suffice?)

Reply:
I changed it to `List[Dict[str, Any]]`. I think this is the right one. Thanks for the review!